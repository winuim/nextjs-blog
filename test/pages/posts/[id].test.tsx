import React from "react";
import { render } from "../../testUtils";
import Post from "../../../pages/posts/[id]";

describe("Home page", () => {
  it("matches snapshot", () => {
    const postData: {
      title: string;
      date: string;
      contentHtml: string;
    } = {
      title: "test tilte",
      date: "2021-01-01",
      contentHtml: "test content",
    };
    const { asFragment } = render(<Post postData={postData} />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
