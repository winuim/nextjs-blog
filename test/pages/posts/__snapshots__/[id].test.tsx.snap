// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`id page matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <header
      class="header"
    >
      <a
        href="/"
      >
        <img
          alt="[Your Name]"
          class="headerImage borderCircle"
          src="/images/profile.jpg"
        />
      </a>
      <h2
        class="headingLg"
      >
        <a
          class="colorInherit"
          href="/"
        >
          [Your Name]
        </a>
      </h2>
    </header>
    <main>
      <article>
        <h1
          class="headingXl"
        >
          test tilte
        </h1>
        <div
          class="lightText"
        >
          <time
            datetime="2021-01-01"
          >
            January 1, 2021
          </time>
        </div>
        <div>
          test content
        </div>
      </article>
    </main>
    <div
      class="backToHome"
    >
      <a
        href="/"
      >
        ‚Üê Back to home
      </a>
    </div>
  </div>
</DocumentFragment>
`;
